experimental/ModelBuilder/MemRefUtils.h

```diff
@@ -124,6 +113,18 @@ makeStridedMemRefDescriptor(void *ptr, void *alignedPtr,
   return descriptor;
 }
 
+// Mallocs a StridedMemRefDescriptor<T, N>* that matches the MLIR ABI.
```

Comment:
Wait how does moving this fix it?

Reply:
Moved it after the function it calls and added the missing template parameters.